dataworks-aws-ucfs-stub.tf

```diff
@@ -0,0 +1,77 @@
+resource "github_repository" "dataworks-aws-ucfs-stub" {
```

Comment:
all Terrform resources should be separated by underscores instead of hyphens
I know there are many examples in this repo with hyphens, but the standard is now defined in the README.md

> Update the github_repository resource name (example) using underscores as separators if required. Its name should use hyphen separators if required, and description attributes should be a single sentence.

Reply:
done